[RISCV] Bump the version of Zicfilp/Zicfiss to 1.0

yetingk opened the pull request:
Both of them are ratified now.
https://wiki.riscv.org/display/HOME/Ratified+Extensions

This patch does not set them to non-experimental, since Zicfilp lacks
lld support and Zicfiss also lacks compiler-rt/libunwind support.

topperc commented:
> Both of them are ratified.
https://wiki.riscv.org/display/HOME/Ratified+Extensions

Experimental is not the same not ratified. Moving an extension out of experimental should mean that the compiler generated code for a it is well tested and complete. Can you summarize the current state of Zicfilp and Zicfiss?

Commit:
[RISCV] Bump the version of Zicfilp/Zicfiss to 1.0

Both of them are ratified now.
https://wiki.riscv.org/display/HOME/Ratified+Extensions

This patch does not set them to non-experimental, since Zicfilp lacks
lld support and Zicfiss also lacks compiler-rt/libunwind support.

topperc reviewed (approved):
LGTM

topperc reviewed (changes requested):
Update ReleaseNotes.rst

topperc commented:
> Since both of them should be experimental now, the pr is only change the version of them now. Hence, I didn't change `llvm/docs/ReleaseNotes.rst`.

Still need to note the version number change

Commit:
Update llvm/docs/ReleaseNotes.rst.

topperc reviewed (approved):
LGTM
